src/scripts/libraryMenu.js

```diff
@@ -387,7 +387,7 @@ import 'flexStyles';
             pageIds: ['mediaLibraryPage', 'librarySettingsPage', 'libraryDisplayPage', 'metadataImagesConfigurationPage', 'metadataNfoPage'],
             icon: 'folder'
         }, {
-            name: globalize.translate('TabPlayback'),
+            name: globalize.translate('TitlePlayback'),
```

Comment:
Why not rename it simply "Playback"?

Reply:
There wasn't a duplicate string called `Playback`, only these two.

Reply:
But shouldn't we rename it to `Playback` at some point, just to make it clearer? (It's not blocking though, to be clear :p )

Reply:
I definitely agree with that, I just didn't want to affect too much with this pull request.